doc/reference/storage_cephobject.md

```diff
@@ -0,0 +1,71 @@
+(storage-cephobject)=
+# Ceph Object - `cephobject`
+
+% Include content from [storage_ceph.md](storage_ceph.md)
+```{include} storage_ceph.md
+    :start-after: <!-- Include start Ceph intro -->
+    :end-before: <!-- Include end Ceph intro -->
+```
+
+{abbr}`Ceph Object Gateway)` is an object storage interface built on top of `librados` to provide applications with a RESTful gateway to Ceph Storage Clusters.
+It provides object storage functionality with an interface that is compatible with a large subset of the Amazon S3 RESTful API.
+
+## Terminology
+
+% Include content from [storage_ceph.md](storage_ceph.md)
+```{include} storage_ceph.md
+    :start-after: <!-- Include start Ceph terminology -->
+    :end-before: <!-- Include end Ceph terminology -->
+```
+
+A *Ceph Object Gateway* consists of several OSD pools and one or more `radosgw` processes that provide object gateway functionality.
```

Comment:
Please can you help me to understand where we are using the emphasis `*` characters? Are we using them for all named concepts?

Reply:
Yes, exactly. It's a common convention to use italics for introducing terms - and since this is a terminology section, it makes sense to emphasis the concepts that we're introducing.

Reply:
Right OK, but we then don't emphasise those same terms elsewhere, only in terminology section?

Reply:
Right, only where we introduce them.

Reply:
Got you, thats clear now thanks.